docker/Makefile

```diff
@@ -26,8 +26,8 @@ generate: env-file-exists build
 	@echo -e "\nSee the getting started docs for information on how to define which teams are allowed to access which servers"
 
 # Start the CA chatbot in the background
-serve: env-file-exists ca-key-exists
-	docker run -d --restart unless-stopped -v $(CURDIR)/example-keybaseca-volume:/mnt:rw ca:latest ./entrypoint-server.sh
+serve: env-file-exists ca-key-exists build
```

Comment:
Is there any reason to add `build` back as a dependency for `serve`? Or just a merge change that got accidentally included?

Reply:
accident, thanks for catching that